bin/make-release.sh

```diff
@@ -43,10 +43,16 @@ find vendor/ -type f -name "*.php" \
  | xargs -L1 git add -f
 find vendor/ -type f -name LICENSE | xargs -L1 git add -f
 find vendor/ -type f -name '*.json' | xargs -L1 git add -f
-find asset/ -type f | xargs -L1 git add -f
+find -L asset/ -type f | xargs -L1 git add -f
```

Comment:
Why is this necessary? The assets are copied over, not symlinked.

Reply:
You're right! Maybe I was using an older version with this change?